src/gentropy/colocalisation.py

```diff
@@ -40,7 +41,7 @@ def __init__(
         credible_set = (
             StudyLocus.from_parquet(
                 session, credible_set_path, recursiveFileLookup=True
-            ).filter(col("finemappingMethod") == "SuSie")
+            ).filter(col("finemappingMethod").isin("SuSie", "SuSiE-inf"))
```

Comment:
Ideally we want to have all finemapping methods inside a directory so that we could import them and not hard code these values. We did something similar for coloc https://github.com/opentargets/gentropy/blob/bc1a1124b0744895f2ce246c23b62861ea600288/src/gentropy/colocalisation.py For the future